src/main/java/uk/ac/ebi/ampt2d/commons/accession/rest/ErrorMessage.java

```diff
@@ -25,28 +25,28 @@
 
     private long timestamp;
 
-    private int status;
+    private HttpStatus status;
```

Comment:
This doesn't get serialized correctly to a rest message, that is why it was a int value.